fiftyone/core/clips.py

```diff
@@ -86,13 +91,15 @@ def __init__(
         self._clips_stage = clips_stage
         self._clips_dataset = clips_dataset
         self.__stages = _stages
+        self.__name = _name
 
     def __copy__(self):
         return self.__class__(
             self._source_collection,
             deepcopy(self._clips_stage),
             self._clips_dataset,
             _stages=deepcopy(self.__stages),
+            _name=self.__name,
```

Comment:
Included `name` here because it makes sense to me that copying a saved view should still be the same saved view